### PR TITLE
fix: Remove the console error

### PR DESCRIPTION
**Pull request by fuchunlan**
Signed-off-by: lannyfu <lannyfu@yunify.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes ##1903

**Special notes for reviewers**:
```
```

**Additional documentation, usage docs, etc.**:

https://user-images.githubusercontent.com/63338728/118946032-9b78ad00-b988-11eb-83a8-1020660c4299.mov

```docs

```

### CODE REVIEW

File: src/actions/configmap.js [hunk 1]
```diff
@@ -87,6 +87,7 @@ export default {
           set(data, 'metadata.resourceVersion', detail.resourceVersion)
           store.update(detail, data).then(() => {
             Modal.close(modal)
+            Notify.success({ content: `${t('Created Successfully')}` })
```
Comment (JohnNiang) [inline src/actions/configmap.js:90]: Actually, notification message should be `Updated Successfully` instead of `Created Successfully`.

**Review by JohnNiang — CHANGES_REQUESTED** [review]

**Comment by LinuxSuRen** [comment]
hi @fuchunlan So glad to see your contribution to this project. But I found a very small detail about your contribution that you might not notice it before. As you can see from the following screenshot, it might be difficult for others to know who you are and what did you do due to the mismatch between your email `lannyfu@yunify.com` and your GitHub email setting. If you want to fix it, please visit [this page](https://github.com/settings/emails).

image

**Comment by fuchunlan** [comment]
Thank you for reminding me.I have added a new email.@LinuxSuRen

**Pull request merged**